doc/source/data/api/input_output.rst

```diff
@@ -226,6 +226,7 @@ Built-in Datasources
    datasource.RangeDatasource
    datasource.TFRecordDatasource
    datasource.MongoDatasource
+   datasource.WebDatasetDatasource
```

Comment:
can you also add a section of `WebDatasetDatasource` for `read_webdataset`? 

Example of image - https://github.com/ray-project/ray/blob/master/doc/source/data/api/input_output.rst?plain=1#L62-L68 .

Reply:
added